docs/developer/advanced/sharing-saved-objects.asciidoc

```diff
@@ -405,8 +405,8 @@ necessary. However, such handling of secondary objects is not considered critica
 ==== 4. What is a "legacy URL alias"?
 
 As depicted above, when an object is converted to become share-capable, if it exists in a non-Default space, its ID gets changed. To
-preserve its old ID, we also create a special object called a _legacy URL alias_ ("alias" for short); this alias retains the target object's
-old ID (_sourceId_), and it contains a pointer to the target object's new ID (_targetId_).
+preserve its old ID, we also create a special object called a <<legacy-url-aliases,_legacy URL alias_>> ("alias" for short); this alias
```

Comment:
I noticed this docs page didn't link to the legacy URL alias docs, so I took this opportunity to add a link. (this docs page predates the other)